packages/components/organisms/f-content-cards/src/components/cardTemplates/homePromotionCard/HomePromotionCard2.vue

```diff
@@ -118,14 +118,18 @@ export default {
         display: block;
         width: 100%;
         box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
-        border-radius: $post-order-card-radius;
-        padding: spacing(x3) calc(35% + 8px) spacing(x3) spacing(x3);
+        border-radius: 4px;
+        padding: spacing(x2) calc(35% + 8px) spacing(x2) spacing(x2);
```

Comment:
Could use the spacing helper inside `calc` here?

```suggestion
        padding: spacing(x2) calc(35% + #{spacing()}) spacing(x2) spacing(x2);
```

Reply:
:heavy_check_mark: